R-packages/evalcast/vignettes/intro-evalcast.Rmd

```diff
@@ -44,87 +48,142 @@ library(evalcast)
-forecast_dates_july <- forecast_dates[forecast_dates >= "2020-07-01" & forecast_dates <= "2020-07-31"]
+forecast_dates <- get_covidhub_forecast_dates("CMU-TimeSeries")
+forecast_dates_july <- forecast_dates[forecast_dates >= "2020-07-01" & 
+                                        forecast_dates <= "2020-07-31"]
 
 # Retrieve past predictions from CovidHub
-predictions_cards_yyg <- get_covidhub_predictions("YYG-ParamSearch", forecast_dates_july)
-predictions_cards_cmu <- get_covidhub_predictions("CMU-TimeSeries", forecast_dates_july)
-
+predictions_cards_ens <- get_covidhub_predictions(
+  "COVIDhub-ensemble", forecast_dates_july, ahead = 3, 
+  signal = "deaths_incidence_num") %>%
+  filter(nchar(geo_value)==2) # remove counties
+predictions_cards_cmu <- get_covidhub_predictions(
+  "CMU-TimeSeries", forecast_dates_july,  ahead = 3, 
+  signal = "deaths_incidence_num")
 # Make new predictions using baseline_forecaster
 predictions_cards <- get_predictions(baseline_forecaster,
                                      name_of_forecaster = "baseline",
                                      signals = signals,
                                      forecast_dates = forecast_dates_july,
                                      incidence_period = "epiweek",
                                      ahead = 3,
+                                     signal_aggregation = "long",
                                      geo_type = "state")
 ```
 
-`baseline_forecaster` created a list of `r length(predictions_cards)` predictions cards, (one for each `(forecast date, ahead)` pair).  Let's look at a predictions card:
+`get_predictions()` and `get_covidhub_predictions()` return a long `data.frame` with `r nrow(predictions_cards)` rows, (one for each `(forecast date, ahead, geo_value, quantile)` combination).
 
 ```{r}
-predictions_cards[[1]]
+predictions_cards_cmu
 ```
 
-Forecasts are probabilistic, meaning that rather than predicting a single value, a distribution of values is generated. For example, the baseline forecaster would have predicted the following distribution in California for `r attributes(predictions_cards_cmu[[1]])$ahead` week(s) after `r attributes(predictions_cards_cmu[[1]])$forecast_date`
+
+Let's look one location:
+
 ```{r}
-pc <- predictions_cards_cmu[[1]]
-ca_fips = covidcast::name_to_fips("California")
-ca_code = substr(ca_fips, 0, 2) # States are identified by a 2 character code
-pc[pc$location == ca_code, ]$forecast_distribution[[1]]
+predictions_cards_cmu %>% 
+  filter(geo_value == "mi", forecast_date == forecast_dates_july[1]) %>%
+  select(geo_value, quantile, value, forecast_date) %>% str()
 ```
 
+Forecasts are generally probabilistic, meaning that rather than predicting a single value, a distribution of values at different quantiles. For example, the Covidhub ensemble forecaster would have predicted the following distribution in California for `r predictions_cards_cmu$ahead[1]` week(s) after `r predictions_cards_cmu$forecast_date[1]`
```

Comment:
I find the phrasing in the first sentence unclear. How about, "Forecasts are generally probabilistic, meaning that they predict a likely distribution of values instead of a single value."? Possibly add another sentence about what quantiles are if that needs to be defined.